impexp-config/src/main/java/org/citydb/config/project/kmlExporter/KmlExporter.java

```diff
@@ -94,6 +93,9 @@
 		"pathOfGltfConverter",
 		"notCreateColladaFiles",
 		"embedTexturesInGltfFiles",
+		"exportGltfBinary",
+		"enableGltfDracoCompression",
+		"exportGltfV1",
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).